### PR TITLE
[CHK-1375] Repository creation and project settings

### PR DESCRIPTION
**Pull request by ciuffagianluca**
Set project file, directories and gradle files

#### List of Changes

Add folder structure for pagopa/pagopa-wallet-service

#### Motivation and Context

Creation of the repository for the wallet-service project

#### How Has This Been Tested?

Tests were performed locally and test the gradle build and the application start

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by infantesimone** [comment]
> This PR exceeds the recommended size of 400 lines. Please make sure you are NOT addressing multiple issues with one PR. _Note this PR might be rejected due to its size._

@giovanniberti  we were talking about yesterday, an action on the PR LOC :-)

File: src/main/java/it/pagopa/wallet/config/RequestFilter.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import org.springframework.web.server.ServerWebExchange;
+import org.springframework.web.server.WebFilter;
+import org.springframework.web.server.WebFilterChain;
+import reactor.core.publisher.Mono;
+import reactor.util.context.Context;
+
+@Component
+@Order(Ordered.HIGHEST_PRECEDENCE)
+@Slf4j
+public class RequestFilter implements WebFilter {
```
Comment (pietro-tota) [inline src/main/java/it/pagopa/wallet/config/RequestFilter.java:18]: Why this the usage of this filter?

**Review by pietro-tota — COMMENTED** [review]
Add spotless plugin with the same formatter file as used for ecommerce modules, such as transactions-service. See https://github.com/diffplug/spotless/tree/main/plugin-gradle#eclipse-jdt

**Review by infantesimone — APPROVED** [review]
sounds good for now 🚀 just remove/comment the webFilter as indicated by @pietro-tota

File: src/main/java/it/pagopa/wallet/config/RequestFilter.java (hunk 1 above)
Comment (ciuffagianluca) [reply src/main/java/it/pagopa/wallet/config/RequestFilter.java:18]: No use for now. It could be useful in further features. But i can remove it.

**Review by infantesimone — APPROVED** [review]

**Pull request merged**